docs/api/ref/Errors.md

```diff
@@ -415,7 +555,34 @@ Not implemented
 ```
 
 </TabItem>
+<TabItem value="golang" label="Go">
+
+```go
+var BridgingError = struct {
+	// Experimental: Returned if a Desktop Agent did not return a response, via 
+  // Desktop Agent Bridging, within the allotted timeout.
+  ResponseTimedOut string
+  // Experimental: Returned if a Desktop Agent that has been targeted by a 
+  // particular request has been disconnected from the Bridge before a 
+  // response has been received from it.
+	AgentDisconnected string
+  // Experimental: Returned for FDC3 API calls that are specified with
+  // arguments indicating that a remote Desktop agent should be targeted
+  // (e.g. raiseIntent with an app on a remote DesktopAgent targeted), 
+  // when the local Desktop Agent is not connected to a bridge. 
+  NotConnectedToBridge string
+  // Experimental: Returned if a message to a Bridge deviates from the schema
+  // for that message sufficiently that it could not be processed.
+  MalformedMessage string
+}{
+	ResponseTimedOut: "ResponseTimedOut",
+	AgentDisconnected:      "AgentDisconnected",
+  NotConnectedToBridge: "NotConnectedToBridge",
+  MalformedMessage: "MalformedMessage",
+}
```

Comment:
```suggestion
var BridgingError = struct {
	// Experimental: Returned if a Desktop Agent did not return a response, via 
	// Desktop Agent Bridging, within the allotted timeout.
	ResponseTimedOut string
	// Experimental: Returned if a Desktop Agent that has been targeted by a 
	// particular request has been disconnected from the Bridge before a 
	// response has been received from it.
	AgentDisconnected string
	// Experimental: Returned for FDC3 API calls that are specified with
	// arguments indicating that a remote Desktop agent should be targeted
	// (e.g. raiseIntent with an app on a remote DesktopAgent targeted), 
	// when the local Desktop Agent is not connected to a bridge. 
	NotConnectedToBridge string
	// Experimental: Returned if a message to a Bridge deviates from the schema
	// for that message sufficiently that it could not be processed.
	MalformedMessage string
}{
	ResponseTimedOut: "ResponseTimedOut",
	AgentDisconnected:      "AgentDisconnected",
	NotConnectedToBridge: "NotConnectedToBridge",
	MalformedMessage: "MalformedMessage",
}
```